tests/test_utils.py

```diff
@@ -1516,7 +1518,7 @@ def test_is_under_replicated_critical():
 
 
 def test_deploy_blacklist_to_constraints():
-    fake_deploy_blacklist = [["region", "useast1-prod"], ["habitat", "fake_habitat"]]
+    fake_deploy_blacklist = [("region", "useast1-prod"), ("habitat", "fake_habitat")]
```

Comment:
Are peoples yamls going to continue to work if they are using [["foo", "bar"]] ? If not, then why did these need to be changed to an array of tuples?

Reply:
The function under test still works with 2-element arrays; mypy does nothing at runtime. This code needed to be changed because test_utils.py is one of the files we're checking with mypy.